### ISSUE TITLE
[Feature Request] User Specified Color Themes

### ISSUE THREAD

**Issue by bryantkelley** [zjs81/meshcore-open#76]
Users should be able to select color themes based on system colors. 
* On Android this means adopting the DynamicColors API so that users can pass their wallpaper colors through to the app.
* On Apple platforms, this means adopting the Colors API and providing system colors as options. Allow the user to select different color themes for light and dark modes.
